solve_test.go

```diff
@@ -47,6 +49,8 @@ func overrideIsStdLib() {
 	isStdLib = func(path string) bool {
 		return false
 	}
+	// NOTE(narqo): this is an ugly hack! One have to think about better way to do cross-package mocking.
+	pkgtree.MockIsStdLib()
```

Comment:
Yknow...I think maybe we could just expose `var IsStdlib` on the subpackage, then have tests overwrite it in much the way they do now. As long as there's a doc on it indicating its intended use, and that overwriting it except for test purposes will cause undefined behavior, then I'm not too bothered.